src/components/NavBar/SearchBar.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import { NavLink } from "react-router-dom";
+import "./SearchBar.css";
+import logo from "../../src/logo.svg"
+import icon from "../../src/icon.svg"
```

Comment:
Path for logo and icon are not found.
```suggestion
import icon from "'../../logo.svg'"
```